src/filters/in_route_repair.c

```diff
@@ -363,7 +363,7 @@ void routein_queue_repair(ROUTEInCtx *ctx, GF_ROUTEEventType evt, u32 evt_param,
 		return;
 	}
 	//TODO, pass any repair URL info coming from broadcast
-	if (!ctx->repair_url) {
+	if (!ctx->repair_urls.nb_items) {
```

Comment:
rather check the presence of repair_servers list for future cases where repair URLs will be populated from the broadcast